patina_dxe_core/Cargo.toml

```diff
@@ -61,3 +61,21 @@ default = []
 std = ["patina/std"]
 doc = ["patina_internal_cpu/doc"]
 compatibility_mode_allowed = []
+
+# Resource Descriptor HOB Version Support
+# =======================================
+# Controls which ResourceDescriptor HOB formats are supported by the DXE Core.
+#
+# Default behavior (no flags):
+#   - Process V2 ResourceDescriptor HOBs with cache attributes support
+#   - Log informational messages when V1 HOBs are encountered
+#   - V1 HOBs are processed for backward compatibility
+#
+# v1_resource_descriptor_support = true:
+#   - Process ONLY V1 ResourceDescriptor HOBs (legacy platforms)
+#   - Skip V2 HOBs completely with warning messages
+#   - No cache attributes support (V1 limitation)
+#
+# This design provides clean separation between V1 and V2 code paths
+# while maintaining shared GCD processing logic for both versions.
```

Comment:
A comment is good here (and please also update the documentation to mention this), but this comment is very verbose and not accurate (V2 only doesn't also process V1 HOBs). Please condense this into a smaller, accurate comment.